ntoskrnl/ps/query.c

```diff
@@ -21,8 +21,9 @@ ULONG PspTraceLevel = 0;
 
 NTSTATUS
 NTAPI
-PsReferenceProcessFilePointer(IN PEPROCESS Process,
-                              OUT PFILE_OBJECT *FileObject)
+PsReferenceProcessFilePointer(
```

Comment:
Could you also update this file too?
https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/include/internal/ps.h;hb=f3e742ddef345eb04d9b4f0646ea66adb80504a6#l144